cypress-tests/cypress/e2e/happyPath/platform/commonTestcases/Onboarding_User.cy.js

```diff
@@ -0,0 +1,1054 @@
+      cy.get(usersSelector.editUserDetailsButton).click();
+      cy.get('[data-cy="user-group-select"]>>>>>').eq(0).type("Admin");
+      cy.wait(1000);
+      cy.get('[data-cy="group-check-input"]').eq(0).check();
+  
+      cy.get(commonSelectors.cancelButton).click();
+  
+      cy.get(usersSelector.userActionButton).click();
+      cy.get(usersSelector.editUserDetailsButton).click();
+      cy.get('[data-cy="user-group-select"]>>>>>').eq(0).type("Admin");
+      cy.wait(1000);
+      cy.get('[data-cy="group-check-input"]').eq(0).check();
+  
+      cy.get(usersSelector.buttonInviteUsers).click();
+      cy.get('.modal-footer > [data-cy="cancel-button"]').click()
+  
+      cy.get(usersSelector.userActionButton).click();
+      cy.get(usersSelector.editUserDetailsButton).click();
+      cy.get('[data-cy="user-group-select"]>>>>>').eq(0).type("Admin");
+      cy.wait(1000);
+      cy.get('[data-cy="group-check-input"]').eq(0).check();
+  
+      cy.get(commonSelectors.cancelButton).click();
+  
+      cy.get(usersSelector.userActionButton).click();
+      cy.get(usersSelector.editUserDetailsButton).click();
+      cy.get('[data-cy="user-group-select"]>>>>>').eq(0).type("Admin");
+      cy.wait(1000);
+      cy.get('[data-cy="group-check-input"]').eq(0).check();
+  
+      cy.get(usersSelector.buttonInviteUsers).click();
+      cy.get('[data-cy="confim-button"]').click()
+  
+      cy.verifyToastMessage(
+        commonSelectors.toastMessage,
+        "User has been updated"
+      );
+  
+      searchUser(data.email);
+      cy.get('[data-name="role-header"] [data-cy="group-chip"]').should("have.text", "Admin");
+    });
+  
+    it("Should verify exisiting user invite flow", () => {
+      data.firstName = fake.firstName;
+      data.email = fake.email.toLowerCase().replaceAll("[^A-Za-z]", "");
+      const workspaceName = data.firstName.toLowerCase();
+  
+      addNewUser(data.firstName, data.email);
+      logout();
+  
+      cy.defaultWorkspaceLogin();
+      cy.apiCreateWorkspace(workspaceName, workspaceName);
+      cy.visit(workspaceName);
+  
+      navigateToManageUsers();
+      fillUserInviteForm(data.firstName, data.email);
+      cy.get(usersSelector.buttonInviteUsers).click();
+      cy.wait(2000);
+      visitWorkspaceInvitation(data.email, workspaceName);
+      cy.wait(3000);
+      cy.clearAndType(onboardingSelectors.Loginemailinput, data.email);
+      cy.clearAndType(onboardingSelectors.LoginpasswordInput, "password");
+      cy.get(onboardingSelectors.signInButton).click();
+      cy.get(usersSelector.acceptInvite).click();
+      cy.verifyToastMessage(commonSelectors.toastMessage, usersText.inviteToast);
+      logout();
+  
+      cy.defaultWorkspaceLogin();
+      navigateToManageUsers();
+      searchUser(data.email);
+      cy.contains("td", data.email)
+        .parent()
+        .within(() => {
+          cy.get("td small").should("have.text", usersText.activeStatus);
+        });
+    });
+  });
```

Comment:
1. Separate Password reset functionality into a different test case document.
2. Focus on user onboarding in this set, including:
 - Individual user signup scenarios.
 - User invite scenarios with different cases.
 - Test cases for first user onboarding should be outlined separately.
 - Include scenarios for bulk user onboarding in this section.

Reply:
Fixed